automarker_2/automarker_app/lib/utils.py

```diff
@@ -115,3 +123,15 @@ def run_command(Cls, command):
                 )
 
         return Cls(stdout=stdout, stderr=stderr)
+
+
+def test_runner_expects_code_imports(cls):
+    class NewRunnerCls(cls):
```

Comment:
KISS

Surely there is a simpler way to do inheritance? 

If most things expect import tags then this should be in the base class, then just override the method where needed